src/styles/globalStyle.ts

```diff
@@ -149,8 +149,10 @@ export const GlobalStyle = createGlobalStyle`
 ${reset}
 
 #root, body, html {
-    width: 100%;
-    height: 100vh;
+    /* width: 100%;
+    height: 100vh; */
+    width: 43rem;
+    height: 92.6rem;
```

Comment:
```suggestion
    /* width: 100%;
    height: 100vh; */
    max-width: 43rem;
    max-height: 92.6rem;
```
로 바꾸는게 피그마의 레퍼선스 기기인 14프로맥스보다 작은 픽셀의 기기에 더 잘 대응할 수 있지 않을까 하는 의견!